.github/workflows/cirque.yaml

```diff
@@ -64,6 +64,17 @@ jobs:
               with:
                   path: ${{ env.GITHUB_CACHE_PATH }}
                   key: ${{ runner.os }}-cirque-${{ steps.cirque-key.outputs.val }}
+                  restore-keys: |
+                    ${{ runner.os }}-cirque-
```

Comment:
Is this intended? Not sure I understand what the intention here is...

Reply:
This is a feature of the cache feature of github action, which allows partial match of a cache key.